lib/holidays/date_calculator/lunar_date.rb

```diff
@@ -3,9 +3,9 @@ module DateCalculator
     # Copied from https://github.com/sunsidew/ruby_lunardate
     # Graciously allowed by JeeWoong Yang (https://github.com/sunsidew)
     class LunarDate
-      attr_accessor :year, :month, :day, :is_leap_month
+      attr_accessor :year, :month, :day
 
-      def to_solar(year, month, day, is_leap_month = false, calendar_symbol = :ko)
+      def to_solar(year, month, day, calendar_symbol)
```

Comment:
Unfortunately the argument name _must_ match the exact expected name, in this case `region`. 😦 

Could we change it here? Then I will merge and do whatever necessary updates to make it all work (along with the other PR in `definitions`).